templates/leaderboard/leaderboard.html

```diff
@@ -9,87 +9,9 @@
 {% endblock %}
 
 {% block content %}
+    <h2>Top Investors</h2>
     <div class="center">
-        {% include "leaderboard/includes/list.html" with name='Investors' canvas_id='InvestorChart' %}
-        {% include "leaderboard/includes/list.html" with name='Artists' canvas_id='ArtistChart' %}
+        {% include "leaderboard/includes/list.html" with name='Investors' leaders=top_earned_investors %}
     </div>
-{% endblock %}
-
-{% block extrajs %}
-    <script type="text/javascript" src="{% static "js/vendor/Chart.min.js" %}"></script>
 
-    <script>
```

Comment:
If we're removing all of this, then we should remove all of the context that's not `top_earned_investors`. Are you good with me removing this?

Reply:
Yeah, that's cool